plugins/hm-stack/inc/rest-controller/namespace.php

```diff
@@ -307,3 +319,27 @@ function( $item ) {
 
 	return new WP_REST_Response( $page_generation_time );
 }
+
+/**
+ * Get deploys history of this site.
+ */
+function get_deploys() {
+	// Check our cache first.
+	$data = wp_cache_get( 'deploys', 'hm-stack' );
+	if ( $data !== false ) {
+		return new WP_REST_Response( $data );
+	}
+
+	$stack_data = API\get_deploys();
+
+	// If we've errored out, return a human-friendly message and code.
+	if ( is_wp_error( $stack_data ) ) {
+		// Prevent constant re-fetching in the event of a failure.
+		wp_cache_set( 'deploys', $stack_data, 'hm-stack', 5 * \MINUTE_IN_SECONDS );
+		return get_wp_error_for_hm_stack_return( $stack_data );
+	}
+
+	wp_cache_set( 'deploys', $stack_data, 'hm-stack', 12 * \HOUR_IN_SECONDS );
```

Comment:
Without knowing what this is being used for, I think it'd be in our best interest to lower this from 12 hours to something closer on a scale of minutes. As a user looking for a recent deploy, if I were to not see it in the list after hearing we just had one, I'd be suspect of something not working and reach out. To save us the support requests, it'd probably be good to lower this 😄